packages/service/lib/plugins/sandbox-wrapper.js

```diff
@@ -27,7 +27,13 @@ module.exports = fp(async function (app, opts) {
         ...patternOptions
       })
     } else {
-      let loaded = await import(`file://${plugin.path}`)
+      let url = ''
+      if (plugin.path) {
+        url = `file://${plugin.path}`
+      } else {
+        url = plugin.module
```

Comment:
I think we should change `createRequire` and use `require.resolve`, otherwise the module won't be loaded?

Reply:
I have done some debugging, the module is loaded, but the decorator `compress` is not on `reply`.
If I go into `@fastify/compress/index.js` and add `fastify.decorate('compress', 'something')` instead of   `  fastify.decorateReply('compress', null)` then `compress` is defined